website/docs/eightshift-libs.md

```diff
@@ -3,31 +3,31 @@ id: eightshift-libs
 title: Eightshift Libs
 ---
 
-[![docs-source](https://img.shields.io/badge/version--4.0.0-eigthshift--libs-blue?style=for-the-badge&logo=php&labelColor=2a2a2a)](https://github.com/infinum/eightshift-libs)
+[![docs-source](https://img.shields.io/badge/source-eigthshift--libs-blue?style=for-the-badge&logo=php&labelColor=2a2a2a)](https://github.com/infinum/eightshift-libs)
 
 This library is aimed at bringing modern back-end development tools to the [Eightshift Boilerplate](https://github.com/infinum/eightshift-boilerplate) or [Eightshift Boilerplate Plugin](https://github.com/infinum/eightshift-boilerplate-plugin), but you can use it on any WordPress project.
 
 It uses a dependency injection container that provides a way to inject dependencies into classes to obey the single responsibility principle (SRP). Every class is responsible for registering its hooks, loaded automatically by the dependency injection container. This provides a more testable environment for your project.
 
-We provide some helpers, abstract classes, interfaces, and abstractions on original WordPress functionality to help you write more modern code.
+We provide helpers, abstract classes, interfaces, and abstractions on original WordPress functionality to help you write more modern code.
 
 ## Prerequisites
 
-In our projects, we use object-oriented PHP, design patterns, and the SOLID principle. When you work on complex projects with various third-party integrations, it makes sense to use these. You want your code to be easily extended, testable, and future-proof.
+In our projects, we use object-oriented PHP, design patterns, and the SOLID principle. When you work on complex projects with various third-party integrations, it makes sense to use the best tools available. You want your code to be easily extended, testable, and future-proof.
```

Comment:
> the best tools available

That's a **bold** statement 😅

Reply:
I'm not implying that our BP is the best tool possible  but rather than all the things listed in the previous sentence

Reply:
Again, the best tools available... If our tooling was more famous, this would be such a beautiful statement to start a flame war on hackernews or reddit 😂